app/main/views/signup.py

```diff
@@ -128,6 +129,12 @@ def create_application(token):
         application = data_api_client.create_application({'status': 'saved'})
         id = application['application']['id']
 
+        notification_message = 'By: {} ({})'.format(
+            current_user.name,
+            current_user.email_address
+        )
+        notify_team('A new seller has started an application', notification_message)
```

Comment:
yeah there are 2 cases here though - 
- creating a new application and user
- create a user for an existing application

I wanted to notify on create application only